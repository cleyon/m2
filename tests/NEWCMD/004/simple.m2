@newcmd pitch{name}{item}
Hello @name@, I just got a great deal on this new @item@ !!!
@endcmd
@pitch{Joe}{air_fryer}
@pitch{Bob}{metal_detector}
