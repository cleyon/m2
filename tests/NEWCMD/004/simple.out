Hello Joe, I just got a great deal on this new air_fryer !!!
Hello Bob, I just got a great deal on this new metal_detector !!!
